### PR TITLE
Add Banner for BackstageCon

### PR DESCRIPTION
**Pull request by taras**
Signed-off-by: Taras Mankovski <taras@frontside.com>

## Hey, I just made a Pull Request!

I noticed that BackstageCon was not visible on the home page. I added a dismissable notification to the home page. 

image

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: microsite/pages/en/index.js [hunk 1]
```diff
@@ -31,6 +31,14 @@ class Index extends React.Component {
                 !
               </div>
             </Banner.Dismissable>
+            <Banner.Dismissable storageKey="2022-03-16-newsletter-cta">
```
Comment (jamieklassen) [inline microsite/pages/en/index.js:34]: maybe a different storage key from the other one
```suggestion
            <Banner.Dismissable storageKey="2022-09-01-backstagecon-cta">
```

Comment (taras) [reply microsite/pages/en/index.js]: Oh right, good call.

File: microsite/pages/en/index.js [hunk 2]
```diff
@@ -31,6 +31,14 @@ class Index extends React.Component {
...
+              <div>
+                🤩 Join us for the first {' '}
+                <a href="https://events.linuxfoundation.org/backstagecon-north-america/">
+                  BackstageCon
+                </a>{' '} on October 24, 2022!
```
Comment (jamieklassen) [inline microsite/pages/en/index.js:39]: prettier asked for this
```suggestion
                🤩 Join us for the first{' '}
                <a href="https://events.linuxfoundation.org/backstagecon-north-america/">
                  BackstageCon
                </a>{' '}
                on October 24, 2022!
```

**Review by freben — APPROVED** [review]

**Comment by freben** [comment]
Thanks!

**Pull request merged**